projects/storefrontlib/src/cms-components/configurator/commons/config-add-to-cart-button/config-add-to-cart-button.component.ts

```diff
@@ -30,16 +30,14 @@ export class ConfigAddToCartButtonComponent implements OnInit {
 
   ngOnInit(): void {
```

Comment:
As there are no inputs, let's get rid of this lifecycle hook, and initialize the properties inline.

Reply:
Also, please make sure all the methods have a return type specified

Reply:
Please remove `public` modified, and consider using `protected` wherever possible (instead of `private`). This will enable clients to override the method, and provide a custom implementation.

Reply:
Please use `protected` instead of `private` in the constructor